### PR TITLE
document NIX_BUILD_SHELL, propagate it in pure build

### PR DESCRIPTION
**Pull request by fkz**
document NIX_BUILD_SHELL in the nix-shell command documentation

see also #730

### CODE REVIEW

File: doc/manual/command-ref/nix-shell.xml [hunk 1]
```diff
@@ -176,6 +176,22 @@ also <xref linkend="sec-common-options" />.</phrase></para>
...
 
 
+<refsection><title>Environment variables</title>
+
+<variablelist>
+
+  <varlistentry><term><envar>NIX_BUILD_SHELL</envar></term>
+    
+    <listitem><para>Shell used to start the interactive environment. 
+    Defaults to the <command>bash</command> found in <envar>PATH</envar></para></listitem>
```
Comment (edolstra) [inline doc/manual/command-ref/nix-shell.xml]: Missing period here.

Comment (fkz) [inline doc/manual/command-ref/nix-shell.xml]: fixed, thanks

**Comment by timbertson** [comment]
To clarify: this propagates the envvar inside a `--pure` shell, and _also_ makes it used by `nix-shell --pure`, right? If so, :+1: !

**Comment by fkz** [comment]
yes

**Comment by edolstra** [comment]
Thanks, applied!

**Pull request closed without merging**